public/.env-template

```diff
@@ -0,0 +1,6 @@
+# Make a copy of this file to .env and update the variables as needed.
+
+# A course creator will be sent to this URL to create a GitHub organization for a course.
+QUICKFEED_ORGANIZATION_URL="https://github.com/organizations/plan"
+# The URL for installing the QuickFeed GitHub app on a course organization.
+QUICKFEED_APP_URL="https://github.com/apps/quickfeed-uis"
```

Comment:
Not sure it makes sense to have this as the default, so perhaps we should remove this default line. But I guess other deployments could still us ours, or am I just not thinking straight... (may well be the case as this hour)

Reply:
I'm not sure if a separate deployment of QuickFeed could use our app. The app is configured with our endpoints, meaning all OAuth logins would go through us. Additionally, their local instance would not have access to use the SCM package without the keys for the app.

Reply:
Ok; then perhaps we should just remove the URL. Or we could leave it in, and just add a comment line that it must be updated for another deployment. Unless you think this might cause problems, if someone forgets to change it...